workspaces/ui-v2/src/components/Panel.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
+import { makeStyles } from '@material-ui/core';
+import { FontFamilyMono } from '<src>/styles';
+
+type PanelProps = {
+  header: React.ReactNode;
+};
+
+export const Panel: FC<PanelProps> = ({ children, header }) => {
+  const classes = useStyles();
+  return (
+    <div className={classes.wrapper}>
```

Comment:
Not something to instantly take action on, but more generally: given the way we're styling things, one really useful little pattern I've found is to accept `className` as a prop for general components like this, and add that to the wrapper div (in addition to the internal one). It allows the user of the component to control things like it's layout, for example child specific flex-box properties. `classnames` is an easy way to join internal css names with one optional one provided through the props.

Reply:
Ah yeah, I'll update this - I added this with some other components but I think I got lazy here